rfc6962-bis.xml

```diff
@@ -444,12 +444,12 @@ Validation Certificates</xref>.
         </section>
         <section title="Redacting Domain Name Labels in Precertificates" anchor="redacting_subdomains">
           <t>
-            When creating a Precertificate, the CA MAY substitute one or more of
-the complete leftmost labels in each DNS-ID with the literal string
-<spanx style="verb">(PRIVATE)</spanx>. For example, if a certificate contains a
+            When creating a Precertificate, the CA MAY substitute each of the
+one or more leftmost labels in each DNS-ID with the label
```

Comment:
Hmm. Scans horribly.

How about "When creating a Precertificate, each label, starting from the left, may be replaced with a ?".